### PR TITLE
Fix & parameterize jdbc tests

### PR DESCRIPTION
**Pull request by onurctirtir**
Parameterize the work done in jdbc tests, add README.md, add driver script

Note: Changes done in `JDBCReleaseTest.java` are majorly picked from `erpre-release-testing` branch, done by @serprex previously

### CODE REVIEW

File: jdbc/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+./run_jdbc_tests.sh <coordinator_port> <jdbc_driver_path>
+```
+
+where,  
+
+coordinator_port: Provide an available port for the coordinator instance of the temporary citus cluster to perform jdbc tests
+jdbc_driver_path: Provide path to jdbc driver for PostgreSQL, (see https://jdbc.postgresql.org/ to download it).
+
+Dependencies:
+* citus_dev
```
Comment (serprex) [inline jdbc/README.md:13]: Should link `[citus_dev](https://github.com/citusdata/tools/tree/develop/citus_dev)`

File: jdbc/run_jdbc_tests.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+jdbc_driver_path=$2
+
+# create & run tmp citus cluster for jdbc test 
+citus_dev stop /tmp/jdbc_test_db --port $coordinator_port
+rm -rf /tmp/jdbc_test_db
+mkdir -p /tmp/jdbc_test_db
+citus_dev make /tmp/jdbc_test_db --port $coordinator_port
+
+# print installed citus version first
+psql -p $coordinator_port -d $USER -c "select citus_version();"
```
Comment (serprex) [inline jdbc/run_jdbc_tests.sh:21]: All cases like `$USER` & `$coordinator_path` should be quoted, `"$USER"` or `"$coordinator_path"`

Comment (hanefi) [reply jdbc/run_jdbc_tests.sh:21]: I suggest you run [ShellCheck](https://github.com/koalaman/shellcheck) (a shell script analysis tool) and try to solve all issues

**Review by gledis69 — APPROVED** [review]
LGTM

**Comment by naisila** [comment]
Fixes #226

**Pull request merged**